src/components/forms/account/AccountForm.tsx

```diff
@@ -125,7 +150,61 @@ export default function AccountForm({ onSave }: AccountFormProps): JSX.Element {
 }
 
 async function onSubmit(data: FormValues, onSave: Function) {
-  await Account.create({ ...data }).save();
-  mutate('/api/accounts');
+  const account = await Account.create({ ...data }).save();
+
+  mutate(
+    '/api/accounts',
+    async (accounts: AccountsMap) => {
+      const [child, parent] = await Promise.all([
+        Account.findOneByOrFail({ guid: account.guid }),
+        Account.findOneByOrFail({ guid: account.parent.guid }),
```

Comment:
we load both accounts like this so childIds and parentId are loaded while parent is not loaded (otherwise we do memory overkill as parent contains an account with another parent, etc).